.env-sample

```diff
@@ -1,4 +1,4 @@
-PORT=9001
+PORT=9000
 PG_CONNECTION_STRING=postgresql://tarpisto:Su5hgVvoqUCRw5vrWmrW@localhost:9876/tarpisto
```

Comment:
Kept giving `Error: connect ECONNREFUSED ::1:5432
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1300:16)` if this was localhost, fixes by changing to `127.0.0.1`.

Reply:
Same thing with `AWS_S3_ENDPOINT`, changing to `127.0.0.1` fixes problems with `localhost`.

Reply:
Are you running inside WSL2?

Reply:
Are you using WSL2?

Reply:
M1 Mac

Reply:
OK, changed localhost -> 127.0.0.1. Does the rest of it work alright?

Reply:
When doing initial testing, rest of it seems to work fine.